README.adoc

```diff
@@ -10,7 +10,7 @@ All OpenShift documentation is sourced in http://www.methods.co.nz/asciidoc/[Asc
 The documentation published from these source files can be viewed at http://docs.openshift.com.
 
 == Contributing to OpenShift documentation
-If you are interested in contributing to OpenShift technical documentation, you can view all our link:../contributing_to_docs[resources] that will help you get set up and provide more information. 
+If you are interested in contributing to OpenShift technical documentation, you can view all our link:./contributing_to_docs[resources] that will help you get set up and provide more information. 
 
```

Comment:
Hmm, perhaps it's possible to remove the leading "./" entirely.
@adellape @CowboysFan @vikram-redhat WDYT?